MSAL/src/native_auth/controllers/sign_up/MSALNativeAuthSignUpController.swift

```diff
@@ -496,32 +517,32 @@ final class MSALNativeAuthSignUpController: MSALNativeAuthBaseController, MSALNa
     private func handleSubmitAttributesResult(
         _ result: MSALNativeAuthSignUpContinueValidatedResponse,
         username: String,
-        signUpToken: String,
+        continuationToken: String,
         event: MSIDTelemetryAPIEvent?,
         context: MSIDRequestContext
     ) -> SignUpSubmitAttributesControllerResponse {
         switch result {
-        case .success(let slt):
-            let state = createSignInAfterSignUpStateUsingSLT(slt, username: username, event: event, context: context)
+        case .success(let continuationToken):
```

Comment:
Here we are shadowing the `continuationToken` parameter, which I think is good, however you don't follow this pattern elsewhere (see other comment). Commenting here too so that we do this consistently across the code.

Reply:
I had to rename the continuationToken in the reset password controller to something different because the new continuation token is optional while the one from the method parameter is not

Reply:
As discussed in the other comment, to avoid confusion all new continuation tokens should be named `newContinuationToken` even when the compiler would allow otherwise.

Reply:
Thanks, variables renamed ;)